python/ml_wrappers/model/predictions_wrapper.py

```diff
@@ -136,9 +156,13 @@ def __init__(self, test_data: pd.DataFrame, y_pred: np.ndarray,
         :type y_pred: np.ndarray
         :param y_pred_proba: Prediction probabilities of the model.
         :type y_pred_proba: np.ndarray
+        :param should_construct_pandas_query: Whether to use pandas query()
```

Comment:
so we want to use pandas by default here or not?  Just want to make sure based on the issue I saw in the if/else above.

Reply:
The choice is to use pandas query in default case so that we don't break the prod with optimization fix. However, for AutoML integration we can set `should_construct_pandas_query` as False to trigger the optimized version.